source/operations/concepts/erasure-coding.rst

```diff
@@ -10,62 +10,131 @@ Erasure Coding
    :local:
    :depth: 2
 
-MinIO Erasure Coding is a data redundancy and availability feature that allows MinIO deployments to automatically reconstruct objects on-the-fly despite the loss of multiple drives or nodes in the cluster. 
-Erasure Coding provides object-level healing with significantly less overhead than adjacent technologies such as RAID or replication. 
+.. meta::
+   :keywords: erasure coding, healing, availability, resiliency
+   :description: Information on MinIO Erasure Coding
 
-MinIO partitions each new object into data and parity shards, where parity shards support reconstruction of missing or corrupted data shards. 
-MinIO writes these shards to a single :ref:`erasure set <minio-ec-erasure-set>` in the deployment.
-MinIO can use either data or parity shards to reconstruct an object, as long as the erasure set has :ref:`read quorum <minio-read-quorum>`.
-For example, MinIO can use parity shards local to the node receiving a request instead of specifically filtering only those nodes or drives containing data shards.
+This page provides a simplified overview of MinIO's implementation of Erasure Coding.
```

Comment:
Wordsmithing idea:
```suggestion
This page provides an introductory overview of MinIO's implementation of Erasure Coding.
```
If you are new to this, it's not exactly easy to understand no matter how its written. Plus there's another `simplified` below, referring to the diagrams.

Reply:
Ooh, good catch. I'd just drop it

```suggestion
This page provides an overview of MinIO's implementation of Erasure Coding.
```

Reply:
Reading through the rendered page, would also be useful to have a glossary link for `erasure coding` here. The Availability and Resiliency page linked below isn't particularly novice-friendly.